Package.swift

```diff
@@ -101,13 +116,20 @@ let package = Package(
       ]
     ),
 
-    // "Support" targets: These contain C family code and are used exclusively
-    // by other targets above, not directly included in product libraries.
+    // "Support" targets: These targets are used exclusively by other targets
+    // above, not directly included in product libraries.
     .target(
       name: "_TestingInternals",
       exclude: ["CMakeLists.txt"],
       cxxSettings: .packageSettings
     ),
+    .target(
+      name: "_TestDiscovery",
+      dependencies: ["_TestingInternals",],
+      exclude: ["CMakeLists.txt"],
+      cxxSettings: .packageSettings,
+      swiftSettings: .packageSettings
```

Comment:
Only needs `swiftSettings:`, not `cxxSettings:`, right?

Reply:
We always need C++ settings for targets that touch `_TestingInternals` or Swift may incorrectly import symbols from C headers.